Rubrik Polaris version 1.0.3 Pull Request (#12543)
demisto/content: Packs/RubrikPolaris/ReleaseNotes/1_0_2.md

#### Integrations
##### Rubrik Radar

> - Modified the graphql query during incident ingestion to filter on the 'lastUpdated' time for new incidents in accordance to backend functionality changes to Radar.  This also fixed a bug where new incidents were not being detected by XSOAR.

Reviewer: ```suggestion
- Improved implementation of the graphql query during incident ingestion to filter on the 'lastUpdated' time for new incidents in accordance to backend functionality changes to Radar.
- Fixed an issue where new incidents were not being detected by XSOAR.
```

> - Added two new options in the Rubrik Polaris integration settings that allow the user to map the Radar Critical and Warning event severity levels to various XSOAR severity levels upon incident fetch.  The default severity mappings are currently:  Radar Critical -> XSOAR HIGH and Radar Warning -> XSOAR LOW.

Reviewer: ```suggestion
- Added the *Radar Critical Severity Level Mapping* and the *Radar Warning Severity Level Mapping* integration parameters in order to allow the mapping of Radar Critical and Warning event severity levels to various XSOAR severity levels upon incident fetch. The default severity mappings are currently:  Radar Critical -> XSOAR HIGH and Radar Warning -> XSOAR LOW.
```

> - Added the 'rubrik-cdm-cluster-connection-state' and 'rubrik-cdm-cluster-location' commands.  These commands have been added in as tasks to the Rubrik Post Intrusion Ransomware Investigation playbook.

Reviewer: ```suggestion
- Added the ***rubrik-cdm-cluster-connection-state*** and ***rubrik-cdm-cluster-location*** commands.  These commands have been added as tasks to the **Rubrik Post Intrusion Ransomware Investigation** playbook.
```

> - Added the Rubrik CDM Cluster Location and Rubrik CDM Cluster Name custom fields.

Reviewer: this is mentioned later - we can remove this line.

Author: This line has been removed.

- Added functionality to pull in the Radar File Change attributes from Radar events during incident ingestion.

> - Various layout changes were made to the Rubrik Radar Incident Summary page.

Reviewer: this is mentioned later - we can remove this line.

Author: These lines have been removed.

> - Renamed the Rubrik Polaris CDM Cluster ID field to Rubrik CDM Cluster ID and modified the associated mapper.

Reviewer: From what I gather you have deprecated the old fields and added new ones - please mention that in the `#### Incident Fields` section below - I'll leave a comment there as well.
please remove this line though

- Added the RubrikRadarFilesAdded, RubrikRadarFilesModified and RubrikRadarFilesDeleted scripts which pulls in the respective Radar File Change attributes.
- Added the RubrikCDMClusterConnectionState script that pulls in the connection status for the CDM cluster.
- Added the RubrikSonarTotalHits script to pull in the total number of Sonar Data Classification Hits.

> - Added the RubrikSonarOpenAccessFiles script to pull the open access permissions file count from Sonar.

Reviewer: These scripts are mentioned later - please remove these lines

> - Added the Rubrik Post Intrusion Ransomware Investigation playbook.  This has been made the default playbook for the Rubrik Radar incident type.

Reviewer: I think we decided to not add that now - please remove this line.

- Added the Rubrik Polaris tab to the Rubrik Radar incident layout.  This new tab includes the Anomaly Detection Event Timeline and Data Classification sections.
- Modified the Radar Mapper to now map the Hostnames field to the Radar Polaris "objectName" field of the Radar event.

> - Removed the Rubrik Analysis Progress, Rubrik Polaris Event Completed, and Rubrik Polaris Last Updated custom fields.

Reviewer: These should be mentioned in their respective sections below. please remove from here.

> - Removed the rubrik_radar_analysis_status command and playbook due to them being no longer needed.

Reviewer: ```suggestion
- Deprecated the ***rubrik-radar-analysis-status*** command as it is no longer needed.
```

- Updated the Docker image to: *demisto/python3:3.9.4.18682*.

#### Scripts
##### New: RubrikCDMClusterConnectionState
- Shows the Rubrik Radar amount of Files Added.

##### New: RubrikRadarFilesAdded
- Shows the Rubrik Radar amount of Files Added.

##### New: RubrikRadarFilesDeleted
- Shows the Rubrik Radar amount of Files Deleted.

##### New: RubrikRadarFilesModified
- Shows the Rubrik Radar amount of Files Modified.

##### New: RubrikSonarOpenAccessFiles
- Shows the Rubrik Polaris Sonar Open Access Files Count.

##### RubrikSonarSensitiveHits
- Shows the data classification hits on an object.

##### New: RubrikSonarTotalHits
- Shows the Rubrik Polaris Sonar Total Hits.

#### Playbooks
##### Rubrik Polaris - Anomaly Analysis

>  - Added the rubrik-cdm-cluster-location and the rubrik-cdm-cluster-connection-state tasks into the playbook under the Sonar Data Classification section.

Reviewer: ```suggestion
 - Added the ***rubrik-cdm-cluster-location*** and the ***rubrik-cdm-cluster-connection-state*** tasks into the playbook under the **Sonar Data Classification** section.
```

#### Mappers
##### Rubrik Polaris Radar - Mapping

> - Varius changes to the mapping file to accomodate new fields.

Reviewer: ```suggestion
- Added various changes to the mapping in order to accommodate the newly added fields.
```

#### Layouts
##### Rubrik Polaris Radar
- Added the Rubrik Polaris tab that contains the Radar event timeline as well as the Data Classification sections.

> - Various layout changes to the Incident Summary tab.

Reviewer: ```suggestion
- Added various layout changes to the Incident Summary tab.
```

#### Incident Fields

> - **Rubrik CDM Cluster ID**
> - **Rubrik CDM Cluster Location**
> - **Rubrik CDM Cluster Name**
> - **Rubrik Polaris Analysis Pipeline**
> - **Rubrik Polaris Analysis Progress**
> - **Rubrik Polaris CDM Cluster ID**
> - **Rubrik Polaris CDM Cluster Name**
> - **Rubrik Polaris Last Updated**
> - **Rubrik Polaris Message**

Reviewer: ```suggestion
- Added the following fields:
   - **Rubrik CDM Cluster ID**
   - **Rubrik CDM Cluster Location**
   - **Rubrik CDM Cluster Name**
   - **Rubrik Polaris Analysis Pipeline**

- Deprecated the following fields:
   - **Rubrik Polaris Analysis Progress**
   - **Rubrik Polaris CDM Cluster ID**
   - **Rubrik Polaris CDM Cluster Name**
   - **Rubrik Polaris Last Updated**
   - **Rubrik Polaris Message**
```

#### Incident Types
##### Rubrik Radar

> - The default integration type was renamed.

Reviewer: ```suggestion
- Updated the default integration type.
```

Author: Thank you @roysagi!  Committed as many of these suggested changes directly here.  For some reason it's not letting me commit the last one directly, so pushed up this commit.
